wandb/sdk/internal/internal_api.py

```diff
@@ -576,7 +576,7 @@ def run_config(self, project, run=None, entity=None):
                 bucket(name: $run) {
                     config
                     commit
-                    files(names: ["wandb-metadata.json", "diff.patch"]) {
+                    files {
```

Comment:
We do support passing in a pattern (which also doesn't create files), but that is resolved using the [LIKE operator](https://github.com/wandb/core/blob/aaffc23943bbd209b13572a95551b6dd6352bdfc/services/gorilla/mysql/metadata.go#L10128), and not a regex. So I don't think that we'd be able to construct a pattern to filter for just the 2 files.

Is getting all the files fine? Or should I add the ability to provide a regex to the server?